apple-xar/Cargo.toml

```
@@ -30,7 +30,7 @@ sha1 = "0.10.6"
 sha2 = "0.10.8"
 signature = { version = "2.2.0", features = ["std"], optional = true }
 thiserror = "1.0.56"
-url = "2.5.0"
+url = "2"
```

What do people think of this change? 

```suggestion
url = ">=2.4.0, <3"
```

My thinking is this carves out an exception for crates using `<=2.5` while still somewhat maintaining semver compatibility.

FWIW I verified locally the workspace compiles with `2.4.0`.